pkgs/tools/misc/coreutils/default.nix

```diff
@@ -53,7 +53,7 @@ stdenv.mkDerivation rec {
       ++ optional aclSupport acl.crossDrv
       ++ optional attrSupport attr.crossDrv
       ++ optionals selinuxSupport [ libselinux.crossDrv libsepol.crossDrv ]
-      ++ optional (stdenv.ccCross.libc ? libiconv)
+      ++ optional (stdenv.ccCross or {} ? libc.libiconv)
```

Comment:
@DavidEGrayson's is nice too. I have no idea whether it works. These sort of "utility attrs" aren't very consistent in nixpkgs, sadly.